README.md

```diff
@@ -1 +1,183 @@
-# catalog-of-my-things
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+
+  <h1><b>Catalog of my things</b></h1>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Catalog of my things <a name="about-project"></a>
+
+**Catalog of my things** is a console interface app that allows the user to store a collection of Items they own. The idea is to set the different attributes for each type of item such as author, genre, labels and more.
+
+## Demo video
+
+[Demo video](https://drive.google.com/file/d/14Pjjj_wD1sgVMaGiGnETCDdcDn9gjAUT/view?usp=sharing)
```

Comment:
Great video presentation, I am really impressed with your work and the presentation.

However, the video is too lengthy. The max length of the video should be 5 mins, briefly introduce yourself, and the project and show a bit of the code and the demo. Don't forget the part of the code that's so cool, which team member 3 did in this video, ensure to include similar in the new video. 

Good luck.